lms/djangoapps/django_comment_client/management/commands/rename_user.py

```diff
@@ -0,0 +1,22 @@
+"""
+Change the username of an existing user
+"""
+
+from django.core.management.base import BaseCommand, CommandError
+from .. import management_utils
```

Comment:
By convention, I _believe_ that we avoid relative imports, so this should be (after moving `management_utils.py` per Jason's feedback):
```python
from django_comment_client import management_utils
```